lux/vislib/matplotlib/Histogram.py

```diff
@@ -0,0 +1,98 @@
+#  Copyright 2019-2020 The Lux Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from lux.vislib.matplotlib.MatplotlibChart import MatplotlibChart
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class Histogram(MatplotlibChart):
+    """
+    Histogram is a subclass of AltairChart that render as a histograms.
+    All rendering properties for histograms are set here.
+
+    See Also
+    --------
+    matplotlib.org
+    """
+
+    def __init__(self, vis, fig, ax):
+        super().__init__(vis, fig, ax)
+
+    def __repr__(self):
+        return f"Histogram <{str(self.vis)}>"
+
+    def initialize_chart(self):
+        self.tooltip = False
+        measure = self.vis.get_attr_by_data_model("measure", exclude_record=True)[0]
+        msr_attr = self.vis.get_attr_by_channel(measure.channel)[0]
+
+        msr_attr_abv = msr_attr.attribute
+
+        if len(msr_attr.attribute) > 17:
+            msr_attr_abv = msr_attr.attribute[:10] + "..." + msr_attr.attribute[-7:]
+
+        x_min = self.vis.min_max[msr_attr.attribute][0]
+        x_max = self.vis.min_max[msr_attr.attribute][1]
+
+        # msr_attr.attribute = msr_attr.attribute.replace(".", "")
+
+        x_range = abs(max(self.vis.data[msr_attr.attribute]) - min(self.vis.data[msr_attr.attribute]))
+        plot_range = abs(x_max - x_min)
+        markbar = x_range / plot_range * 12
```

Comment:
For histograms, we can decrease the bin width, the bars look a bit thick. It seems like `markbar` isn't being used here. maybe matplotlib has a better mechanism for determining histogram bar width, number of bins, and spacing?